com.dynamo.cr/com.dynamo.cr.bob/build.gradle

```diff
@@ -48,23 +48,25 @@ gradle.projectsEvaluated {
         def protoDir = file(proto.dir)
         def protoFilePath = file("${proto.dir}/${proto.file}")
         def protoTaskName = "protoTask_${proto.file.replace('/', '_').replace('.', '_')}" as Object
-
+        tasks.named('compileJava').configure {
+            dependsOn protoTaskName
+        }
         tasks.create(name: protoTaskName, type: Exec) {
             group = 'build'
             inputs.file(protoFilePath)
-            outputs.dir('generated')
+            outputs.dir("${project.bobDir}/generated")
```

Comment:
Make sure that we use folders independently on work directory (when we run tests for example)